### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by austr1an**
if you find other problems in josh api reference, feel free to commit 😉

### CODE REVIEW

File: docs/api-docs.md [hunk 1]
```diff
@@ -54,14 +54,7 @@ Initializes a new Josh, with options.
...
-  provider,
-});
+const Josh = require("@joshdb/core");
const provider = require("@joshdb/sqlite");

// sqlite-based database, with default options
const sqliteDB = new Josh({
  name: 'mydatabase',
  provider,
});
```
Comment (AustrianDev) [inline docs/api-docs.md:57]: wtf i don't know why that displays this shit, but this is line returns actually

Comment (dan-online) [reply docs/api-docs.md:57]: yea I don't know where the actual changes are haha

**Comment by eslachance** [comment]
`api-docs.md` is autogenerated by the script, please take a look at /docs/build.js for that script. The MD output should not be directly modified.

**Comment by AustrianDev** [comment]
I did not modify it, i ran build.js iirc

**Pull request closed without merging**